blablas/ep21/index.md

```diff
@@ -13,14 +13,58 @@ audio:
 
 In this episode of GeeksBlabla, Geeksblabla team give some tips for succeeding in computer science studies.
 
+## Guests
+
+- [Mohammed Aboullaite](http://aboullaite.me/)
+
+- [Youssouf EL Azizi](https://elazizi.com/)
+
+- [Meriem Zaid](https://www.facebook.com/MeriemZaid)
+
+- [Oumaima Makhlouk](https://www.facebook.com/oumi.makhlouk)
+
+- [Oussama Hyad](https://www.facebook.com/heoussama.oussama)
+
 ## Notes
 
-<br/>
-<br/>
-<br/>
+00:00 - Introduction: Welcoming and presenting guests.
+
+00:08 - First year of school experience and some tips.
+
+00:28 - Tips to deal better with the coming years of school and The importance of self-learning.
+
+00:36 - Is that the degree is important to integrate the field of development ?
+
+00:38 - Freelancing while studying.
+
+00:40 - The importance of extracurricular activities in academic and professional careers.
+
+00:50 - Is that the exams score is quite important ?
+
+00:58 - How to find a good end-of-school project (PFE) and prepare for the professional world ?
+
+01:11 - End-of-school project (PFE) in a multinational company Vs Startup.
+
+01:19 - How to prepare for a good defense ?
```

Comment:
Not sure what you mean by `good defense`

Reply:
its mean project presentation "soutenance"

Reply:
Hmm, a little confusing, why not `good project presentation `

Reply:
Yeah, I will modify it